### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by youen**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: make lint
+        run: docker-compose exec -w /workspace vscode make lint
```
Comment (youen) [inline .github/workflows/main.yml:30]:
```suggestion
      - name: make lint
        run: docker-compose up -d
```

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -52,6 +56,9 @@ RUN wget -O- -nv https://install.goreleaser.com/github.com/goreleaser/goreleaser
 ADD https://github.com/ovh/venom/releases/download/v0.27.0/venom.linux-amd64 /bin/venom
 RUN chmod +x /bin/venom
 
+# Add docker-compose
+COPY --from=docker-compose /usr/local/bin/docker-compose /usr/local/bin/docker-compose
 
+RUN useradd -ms /bin/bash vscode
```
Comment (adrienaury) [inline .devcontainer/Dockerfile:62]:
```suggestion
RUN addgroup -g 1000 -S vscode \
 && adduser -S -s /bin/zsh -G vscode -D -u 1000 vscode \
 && cp -r /root/. /home/vscode \
 && chown -R vscode:vscode /home/vscode \
 && apk add -q --update --progress --no-cache su-exec sudo \
 && echo vscode ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/vscode \
 && chmod 0440 /etc/sudoers.d/vscode \
 && echo 'Defaults env_keep += "GOPATH"' >> /etc/sudoers
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+
```
Comment (adrienaury) [inline .github/workflows/main.yml:28]: Somewhere here, add a step
```
chmod -R 1000:1000 .
```

**Review by adrienaury — APPROVED** [review]

**Comment by youen** [comment]
With have some tests fix now :(

```
Total:104 Duration:17.959748727s
OK:89
KO:15
Skipped:0
TestSuite:25
TestCase:120
```

**Pull request merged**